src/App.css

```diff
@@ -0,0 +1,69 @@
+* {
```

Comment:
Since the project is asking you to use SASS, you'll need to change this file (and any other file ending in css) to scss.